tests/pytest_tests/conftest.py

```diff
@@ -26,10 +24,15 @@
 
 
 @pytest.fixture(autouse=True)
-def disable_sentry_in_core(monkeypatch: pytest.MonkeyPatch) -> None:
-    """Disables Sentry for all tests with core."""
+def disable_sentry(monkeypatch: pytest.MonkeyPatch) -> None:
+    """Configures wandb env variables to suitable defaults for tests."""
+    # Don't write to Sentry.
+    monkeypatch.setenv("WANDB_ERROR_REPORTING", "false")
```

Comment:
this one unfortunately has to stay as it was bc Sentry is set up upon wandb import.

Reply:
thank you, that's what i was worried about